Update step-by-step-guide (#1426)
IgniteUI/igniteui-docfx: en/components/general/cli/templates.md

---

> title: Ignite UI for Angular Templates

Reviewer: ```suggestion
title: Component and Scenario CLI Templates
```

> _description: List of all available Ignite UI for Angular templates.

Reviewer: ```suggestion
_description: List of all available CLI templates with Ignite UI for Angular
```

_keywords: Ignite UI for Angular, UI controls, CLI, Templates, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library
---

> ## Ignite UI for Angular Templates

Reviewer: ```suggestion
## Component Templates
```

> The following table describes the Ignite UI Angular components that can be generated using the [`Ignite UI CLI`](https://github.com/IgniteUI/igniteui-cli). In addition you can find links to the available demos in our documentation, but note that those are not exactly identical to the ones generated by the CLI.

Reviewer: ```suggestion
The following table describes the Ignite UI Angular components that can be generated using the [Ignite UI CLI](../cli-overview.md). In addition you can find links to the available demos in our documentation, but note that those are not exactly identical to the ones generated by the CLI.
```

|date-picker        | <code>ig add date-picker newDatePicker</code><details>Basic IgxDatePicker.</details>          | Basic [IgxDatePicker](../../date_picker.md) with one-way data binding.          |
|time-picker        | <code>ig add time-picker newTimePicker</code><details>Basic IgxTimePicker.</details>          | Basic [IgxTimePicker](../../time_picker.md) with initial value set and one-way data binding.           |
|calendar           | <code>ig add calendar newCalendar</code><details>IgxCalendar with single selection.</details> | Basic [IgxDatePicker](../../date_picker.md) with one-way data binding.          |

> |<b>Custom templates</b>  |                                                                                               |

Reviewer: ```suggestion

## Scenario templates
| Template          |  Code and description                                                                         | Demo               |
| ----------------- | --------------------------------------------------------------------------------------------  |------------------- |
```
